tchannel/thrift/client.py

```diff
@@ -75,15 +78,21 @@ def post_comment(articleId, msg, hostport=None):
 
     method_names = get_service_methods(service_module.Iface)
 
-    def new(cls, tchannel, hostport=None, trace=False):
+    def new(cls, tchannel, hostport=None, trace=False, protocol_headers=None):
```

Comment:
Yes we do. You need to update it with sync client. But it's not high priority. You can do that after the coming api call change.